.template/lib/main.dart

```diff
@@ -65,10 +88,55 @@ class MyHomePage extends StatelessWidget {
             Text(
               FlutterConfig.get('SECRET'),
               style: const TextStyle(color: Colors.black, fontSize: 24),
-            )
+            ),
+            const SizedBox(height: 24),
+            ElevatedButton(
+              onPressed: () => context.go('/$routePathPage2'),
+              child: const Text("Navigate to Page 2"),
+            ),
           ],
         ),
       ),
     );
   }
 }
+
+class Page2Screen extends StatelessWidget {
```

Comment:
```suggestion
class SecondScreen extends StatelessWidget {
```

a Page is a reference to a WEB page, while a Screen is more relative to a mobile SCREEN, besides, we should avoid the number in our names. Instead, how about calling by its order, i.e `first` - `second` - `third` in the written format?
Also, do we really need to have a 3rd one? I feel like unnecessary, remember that we should keep this template as simple as possible, right? For demo purposes, you should do it in example/, don't make a mess in this backbone template.

Reply:
@sleepylee agree, I will update 👌

Reply:
Updated 🙏